composition-js/src/merging/merge.ts

```diff
@@ -2798,4 +2823,16 @@ class Merger {
         : err;
     });
   }
+
+  private validateSubscriptionField(sources: FieldOrUndefinedArray) {
+    // no subgraph marks field as @shareable
+    const fieldsWithShareable = sources.filter((src, idx) => src && src.appliedDirectivesOf(this.metadata(idx).shareableDirective()).length > 0);
+    if (fieldsWithShareable.length > 0) {
+      const nodes = sourceASTs(...fieldsWithShareable);
+      this.errors.push(ERRORS.INVALID_FIELD_SHARING.err(
+        `Fields on root level subscription object cannot be marked as shareable`,
```

Comment:
is this temporary thing or we will support it in the future? (I thought the intent was to support this?)

Reply:
I don't think we intend on subgraphs being able to merge their subscriptions in the supergraph. Let me know if I'm missing something.